packages/perseus-editor/src/widgets/dropdown-editor.tsx

```diff
@@ -115,9 +123,41 @@ class DropdownEditor extends React.Component<Props> {
                         </p>
                     </InfoTip>
                 </div>
+                <div className="dropdown-visible-label">
+                    <LabeledTextField
+                        label="Visible label"
+                        value={this.props.visibleLabel}
+                        onChange={this.onVisibleLabelChange}
+                    />
+                    <InfoTip>
+                        <p>Optional visible label</p>
+                    </InfoTip>
```

Comment:
The tooltip icons seem oddly placed (being below the input field instead of after the label text). Instead of using `<LabeledTextField>`, would it be possible to have a separate label, followed by the `<InfoTip>` and then a regular `<TextField>`?

![Tooltip Placement](https://github.com/user-attachments/assets/cfa26d38-a991-448d-abe3-3f3bae3142ec)